docs/en/advance/long_context.md

```diff
@@ -0,0 +1,71 @@
+# Context length extrapolation
```

Comment:
may add to `index.rst` to update this doc in readthedocs

Reply:
OK, added.